### ISSUE TITLE
Bad request while downloading zip

### ISSUE THREAD

**Issue by shivarm** [quarkusio/code.quarkus.io#1231]
While downloading zip got error like 
```
Bad request > Error while rendering template: /codestarts/quarkus/github-action-codestart/base/action.tpl.qute.yml

```

**Expected behavior**
(Describe the expected behavior clearly and concisely.)

**Screenshots**
(If applicable, add screenshots to help explain your problem.)
![image](https://github.com/quarkusio/code.quarkus.io/assets/91419219/c740f92e-9f39-4cd6-9502-bead2f9a7a8f)

**Environment (please complete the following information):**
 - Chrome: 
 - Windows 11:

**Comment by ia3andy** [comment]
@shivam-sharma7 could you give the link of the request?

**Comment by shivam-sharma7** [comment]
> @shivam-sharma7 could you give the link of the request?

@ia3andy oh when i selected other extention then it's downloaded but for [quarkus-github-action] see- https://code.quarkus.io/d?e=io.quarkiverse.githubaction%3Aquarkus-github-action&cn=code.quarkus.io

**Comment by tqvarnst** [comment]
Seems to be an issue with the starter code in the quarkus-github-action extension. If I remove that extension and add resteasy reactive it works.

**Comment by shivam-sharma7** [comment]
@tqvarnst Yes, If you add any other extension then it will work but if you select `quarkus-github-action` then it will not allow you to download the zip.

**Comment by shivam-sharma7** [comment]
@tqvarnst Yes, If you add any other extension then it will work but if you select `quarkus-github-action` then it will not allow you to download the zip.

**Comment by ia3andy** [comment]
@gsmet we still have this issue, what should we do?

**Comment by gsmet** [comment]
I should probably unlist the extension from code.quarkus.io for the time being. But I don't think we can unlist it only from there, it would be removed from extensions.quarkus.io too.

Ideally, we would have a UI that would ask for the additional information that is needed for the codestart (the description and the target GitHub repo), based on the extension metadata (a bit similar to what is done for `worfklow_dispatch` GitHub Actions workflow when you have inputs). The codestart requires some information to properly set up the `action.yml` descriptor.

Everything is documented here: https://docs.quarkiverse.io/quarkus-github-action/dev/create-github-action.html
